test/compflags/bradc/mungeUserIdents/testmunge-extern.chpl

```diff
@@ -4,12 +4,16 @@ extern record bar {
   var foo: real;
 }
 
-extern var baz: bar;
-var b: bar;
-
-foo();
-writeln(baz);
-writeln(baz.foo);
-b.foo = 1.2;
-writeln(b);
+// Use 'main' to ensure that 'bar' is present in the generatec C for this
+// module. If it's global, it will be in the header file where this test's
+// prediff will not look.
```

Comment:
I would argue that implies we should add a test case that checks the header file

Reply:
Could that be part of a separate PR (which I will start right away) just to keep this branch cleaner?

Reply:
I'm okay with that